cmd/alternate_state.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"github.com/qlik-oss/corectl/internal"
+	"github.com/qlik-oss/corectl/printer"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var listAlternateStateCmd = &cobra.Command{
+	Use:     "ls",
+	Args:    cobra.ExactArgs(0),
+	Short:   "Print a list of all alternate states in the current app",
+	Long:    "Print a list of all alternate states in the current app",
+	Example: "corectl state ls",
+
+	Run: func(ccmd *cobra.Command, args []string) {
+		state := internal.PrepareEngineState(rootCtx, headers, certificates, false)
+		items := internal.ListAlternateState(state.Ctx, state.Doc)
+		printer.PrintStates(items, viper.GetBool("bash"))
+	},
+}
+
+var addAlternateStateCmd = &cobra.Command{
+	Use:     "add <alternate-state-name>",
+	Args:    cobra.ExactArgs(1),
+	Short:   "Add an alternate states in the current app",
+	Long:    "Add an alternate states in the current app",
+	Example: "corectl state add NAME-1",
+
+	Run: func(ccmd *cobra.Command, args []string) {
+		stateName := args[0]
+		if stateName == "" {
+			internal.FatalError("no state name specified")
+		}
+		state := internal.PrepareEngineState(rootCtx, headers, certificates, false)
+		internal.AddAlternateState(state.Ctx, state.Doc, stateName)
+		if !viper.GetBool("no-save") {
+			internal.Save(rootCtx, state.Doc)
+		}
+	},
+}
+
+var removeAlternateStateCmd = &cobra.Command{
+	Use:     "rm <alternate-state-name>",
+	Args:    cobra.ExactArgs(1),
+	Short:   "Removes an alternate states in the current app",
```

Comment:
Since you can only remove one state this should be singular

Reply:
Fixed